modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -35,10 +35,35 @@
 #include <openspace/scripting/scriptengine.h>
 #include <ghoul/filesystem/filesystem.h>
 #include <ghoul/logging/logmanager.h>
+#include <scn/scn.h>
 
 namespace {
-    constexpr std::string_view _loggerCat = "SkyBrowserModule";
+constexpr std::string_view _loggerCat = "SkyBrowserModule";
+
+bool browserBelongsToCurrentNode(std::string identifier) {
+    std::string delimiter = "_";
+    size_t found = identifier.find(delimiter);
+    std::string errorMessage = "The Sky Browser encountered a problem when it tried to "
+        "initialize the browser";
+    if (found == identifier.size()) {
+        throw ghoul::RuntimeError(errorMessage);
+    }
+    else {
+        std::string res = identifier.substr(found + 1, identifier.size());
+        if (res == "") {
+            throw ghoul::RuntimeError(errorMessage);
+        }
+        // Convert the last char to an int
+        int nodeId = std::stoi(res);
+        return nodeId == openspace::global::windowDelegate->currentNode();
+    }
+}
 
+std::string getPrunedIdentifer(std::string identifier) {
```

Comment:
For our coding style -> `prunedIdentifier`.  We only use the "get" prefix if the return value is passed in as a reference